### PR TITLE
power-profiles-daemon: add startup dependency

### PR DESCRIPTION
**Pull request by Luciogi**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-Glibc)

### CODE REVIEW

**Comment by Luciogi** [comment]
should I bump version or wait for maintainer?

**Review by ahesford — CHANGES_REQUESTED** [review]
The revision needs to be bumped, or this package won't be rebuilt.

**Comment by Luciogi** [comment]
@ahesford done

File: srcpkgs/power-profiles-daemon/files/power-profiles-daemon/run [hunk 1]
```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
 exec 2>&1
+sv start dbus >/dev/null || exit 1
```
Comment (ahesford) [inline srcpkgs/power-profiles-daemon/files/power-profiles-daemon/run:3]: This is supposed to be `sv check`, not `sv start`.

**Comment by Luciogi** [comment]
ah sorry.
Now Fixed
@ahesford

**Pull request merged**